<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataList</class>
 <widget class="QWidget" name="DataList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>373</width>
    <height>223</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">DataList</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="layoutButtons">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="AcceptRejectButtons" name="buttonsAcceptReject" native="true"/>
     </item>
     <item>
      <widget class="ToolInterpolate" name="toolInterpolate" native="true"/>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QComboBox" name="comboTimeStep">
       <property name="toolTip">
        <string>Time step for times to be shown in the data list.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkFilterTimes">
       <property name="toolTip">
        <string>If checked, filter obstimes from the database with the time step.
If unchecked, show all times in the range even if not in the database.</string>
       </property>
       <property name="text">
        <string>Filter</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonSaveAs">
       <property name="toolTip">
        <string>Save selected (or all) data to a spreadsheet file in CSV format.</string>
       </property>
       <property name="text">
        <string>Saveâ€¦</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="DataListTable" name="table"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DataListTable</class>
   <extends>QTableWidget</extends>
   <header>hqc/DataList.hh</header>
  </customwidget>
  <customwidget>
   <class>AcceptRejectButtons</class>
   <extends>QWidget</extends>
   <header>common/gui/AcceptRejectButtons.hh</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ToolInterpolate</class>
   <extends>QWidget</extends>
   <header>common/gui/ToolInterpolate.hh</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonSaveAs</sender>
   <signal>clicked()</signal>
   <receiver>DataList</receiver>
   <slot>onButtonSaveAs()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>371</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>15</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboTimeStep</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>DataList</receiver>
   <slot>onUITimeStepChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>151</x>
     <y>11</y>
    </hint>
    <hint type="destinationlabel">
     <x>189</x>
     <y>30</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkFilterTimes</sender>
   <signal>toggled(bool)</signal>
   <receiver>DataList</receiver>
   <slot>onCheckFilter(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>206</x>
     <y>9</y>
    </hint>
    <hint type="destinationlabel">
     <x>105</x>
     <y>29</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onButtonSaveAs()</slot>
  <slot>onAcceptQC2()</slot>
  <slot>onAccept()</slot>
  <slot>onReject()</slot>
  <slot>onRejectQC2()</slot>
  <slot>onButtonResetColumns()</slot>
  <slot>onUITimeStepChanged(int)</slot>
  <slot>onCheckFilter(bool)</slot>
 </slots>
</ui>
